docs/src/systems/implicit_incompressible_sph.md

```diff
@@ -0,0 +1,233 @@
+# [Implicit Incompressible SPH](@id iisph)
+
+Implicit Incompressible SPH as introduced by [M. Ihmsen](@cite ihmsen2013) is a scheme that computes pressure values, by iteratively solving a linear system using a relaxed jacobi system, to resolve the particles density deviation from the reference density. This method solves a linear system $Ax=b$ where the pressure values that are used for the pressure acceleration are the unknown variable $x$.
+It does not use a state equation to generate pressure values like the [weakly compressible SPH scheme](weakly_compressible_sph.md).
+
+
+To derive the formulation for the linear system we start by discretizing the Continuity equation
+
+```math
+\frac{D\rho}{Dt} = - \rho \nabla \cdot v
+```
+
+For a particle $i$ a forward difference method is used to discretizie the left hand side of the equation to get
+
+```math
+\frac{\rho_i(t+ \Delta t - \rho_i(t))}{\Delta t}.
+```
+
+The right-hand side is discretized by the difference formulation of the divergence of the velocity field.
+
+```math
+\nabla \cdot \textbf{v}_i = \frac{1}{\rho_i} \sum_j m_j \textbf{v}_{ij} \nabla W_{ij},
+```
+where $\textbf{v}_{ij}$ is equal to $\textbf{v}_i - \textbf{v}_j$.
```

Comment:
````suggestion
The divergence in the right-hand side is discretized with the SPH discretization for particle ``i`` as
```math
-\frac{1}{\rho_i} \sum_j m_j \textbf{v}_{ij} \nabla W_{ij},
```
where ``\bm{v}_{ij} = \bm{v}_i - \bm{v}_j``.
````
1. The minus sign is important when discretizing a divergence.
2. Use `\bm` for math (change everywhere please).
3. Be careful with equations. The RHS is not equal to this, but it is approximated by this discretization.